src/components/Input/__tests__/__snapshots__/CheckBoxInput.spec.js.snap

```diff
@@ -278,6 +278,146 @@ exports[`<CheckBoxInput /> should render the correct markup when additional prop
 />
 `;
 
+exports[`<CheckBoxInput /> should render the correct markup when isTopAligned 1`] = `
+.c0 {
+  outline: none;
+  margin: 0;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  position: relative;
+  background-color: rgba(255,255,255,1);
+  border-color: #999999;
+  border-radius: 2px;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -webkit-justify-content: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
```

Comment:
It looks like your test is not working. It should align items on `flex-start`

Reply:
for the input the only change is a margin only when the size is small in order to align it properly. Added another test for the whole CheckBox component. See - CheckBoxGroup.spec.js.snap